src/api/db/migrate/20160317164117_sanitize_invalid_repository_names.rb

```diff
@@ -0,0 +1,17 @@
+class SanitizeInvalidRepositoryNames < ActiveRecord::Migration
+  def up
+    Repository.where("name REGEXP ?", "^_|[:\/]+").find_each do |repository|
+      repository.name = repository.name.gsub(/^_|[:\/]+/, "")
+      repository.name = "please_enter_a_valid_name_#{SecureRandom.hex(5)}" if repository.blank?
+      repository.save!
```

Comment:
I am removing the migration the. OBS doesn't have any invalid repos right now and other instances would have to be fixed manually to keep backend and frontend data in sync.